src/annotator/sidebar.tsx

```diff
@@ -89,9 +89,9 @@ export function createSidebarIframe(config: SidebarConfig): HTMLIFrameElement {
 
   // In viahtml, pywb uses wombat.js, which monkey-patches some JS methods.
   // One of those causes the `allow` attribute to be overwritten, so we want to
-  // make it non-writable to preserve the permissions we set above.
+  // define a noop setter to preserve the permissions we set above.
```

Comment:
Can you add a note here that links to the upstream pywb fix, so we can remove this hack once that is deployed.